packages/fxa-auth-server/lib/routes/utils/signup.js

```diff
@@ -63,13 +66,14 @@ module.exports = (log, db, mailer, push, verificationReminders) => {
       );
 
       // Our post-verification email is very specific to sync,
-      // so only send it if we're sure this is for sync.
-      if (service === 'sync') {
+      // so only send it if we're sure this is for sync or sync scoped client.
+      const scopeSet = ScopeSet.fromArray(scopes);
+      if (service === 'sync' || scopeSet.intersects(NOTIFICATION_SCOPES)) {
         const onMobileDevice =
           userAgent(request.headers['user-agent']).deviceType === 'mobile';
         const mailOptions = {
           acceptLanguage: request.app.acceptLanguage,
-          service,
+          service: 'sync',
```

Comment:
The email template doesn't really need to know what scopes are involved, only that its sync based.